### PR TITLE
Some cleanup for cmake

### PR DESCRIPTION
**Pull request by cryptomilk**
Some cleanup for cmake.

### CODE REVIEW

File: packaging/windows/BUILD.txt [hunk 1]
```diff
@@ -223,7 +223,6 @@ B) Cross compile on Linux
 
     cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchain_mingw64.cmake \
         -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
-        -DRUNTIME_INSTALL_PREFIX=${RUNTIME_PREFIX} \
```
Comment (TurboGit) [inline packaging/windows/BUILD.txt]: Isn't this breaking cross-compilation?

Comment (cryptomilk) [reply packaging/windows/BUILD.txt]: It isn't used anywhere. It is also not a CMAKE variable.

File: CMakeLists.txt [hunk 2]
```diff
@@ -79,45 +78,6 @@ endif()
...
-  MESSAGE(FATAL_ERROR "The target platform is not supported!")
-endif(NOT IS_SUPPORTED_PLATFORM)
-MESSAGE(STATUS "Is the target platform supported: ${IS_SUPPORTED_PLATFORM}")
-
-set(CMAKE_REQUIRED_INCLUDES "${CMAKE_REQUIRED_INCLUDES_OLD}")
-unset(CMAKE_REQUIRED_INCLUDES_OLD)
-
-# When cross compiling, CMAKE_INSTALL_PREFIX will point to something like "/opt/darktable", but that's not useful when using the path to load
-# modules on runtime. Then we need something like "C:\Program Files\darktable". Doesn't need to be set when doing regular compiles.
-if(NOT DEFINED RUNTIME_INSTALL_PREFIX)
```
Comment (TurboGit) [inline CMakeLists.txt]: It is used here, in part of the code you've removed.

Comment (cryptomilk) [reply CMakeLists.txt]: This will just set a default value if it is not set, but where is it used. I don't find anything.

**Comment by TurboGit** [comment]
Ok, let's see how this fly then. Merging. Thanks.

**Pull request merged**